hack/tools/go.mod

```diff
@@ -3,194 +3,72 @@ module github.com/vmware-tanzu/hack/tools
 go 1.17
 
 require (
-	github.com/golangci/golangci-lint v1.43.0
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.5.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/shuLhan/go-bindata v4.0.0+incompatible
-	golang.org/x/tools v0.1.10
+	golang.org/x/tools v0.1.12
```

Comment:
Have you checked that with `golangci-lint` v1.146.0 the problem with linters being ignored no longer happens with `golang.org/x/tools v0.1.12`?

Reply:
yes I did. The linters ran and generated the same outputs as before the change, except for 3 additional warnings mentioned.